src/BrowserInterop/Extensions/BrowserInteropJsRuntimeExtension.cs

```diff
@@ -151,7 +151,7 @@ public static async ValueTask<WindowInterop> Window(this IJSRuntime jsRuntime)
         public static async ValueTask InvokeInstanceMethod(this IJSRuntime jsRuntime, JsRuntimeObjectRef windowObject,
             string methodName, params object[] arguments)
         {
-            await jsRuntime.InvokeVoidAsync("browserInterop.callInstanceMethod",
+            await jsRuntime.InvokeVoidAsync("browserInterop.callInstanceAction",
```

Comment:
what's the point of this renaming?

Reply:
callInstanceAction is a new method, not a rename. 'Action' indicates that there is no return value, like in .NET the Func / Action delegates. I woud prefer to follow the .NET namings: Function for methods with return value and Action for methods without return value

Reply:
The problem with browserInterop.callInstanceMethod is that it always tries to serialize the return value of the invoke method. But I as a user want to decide whether I need the return value or not, and I do that already by calling ValueTask InvokeInstanceMethod which indicates that I don't expect / don't need the return value. That shall be respected by the javascript side.

Reply:
Ok I see cool